src/index.js

```diff
@@ -1,24 +1,13 @@
-const jsDom = require('jsdom')
+const jsdom = require('jsdom')
+const { JSDOM } = jsdom
 const d3 = require('d3')
 
 module.exports = D3Node
 
 module.exports.d3 = d3
-module.exports.jsDom = jsDom
-
-function fixXmlCase (text) {
-  // Fix a jsdom issue where all SVG tagNames are lowercased:
-  // https://github.com/tmpvar/jsdom/issues/620
-  var tagNames = ['linearGradient', 'radialGradient', 'clipPath', 'textPath']
-  for (var i = 0, l = tagNames.length; i < l; i++) {
-    var tagName = tagNames[i]
-    text = text.replace(
-      new RegExp('(<|</)' + tagName.toLowerCase() + '\\b', 'g'),
-      function (all, start) {
-        return start + tagName
-      })
-  }
-  return text
+module.exports.JSDOM = JSDOM
```

Comment:
this rename breaks existing API

Reply:
If it makes sense to change, thats fine, just need to call it out in the release as a breaking change.